datarepo-clienttests/build.gradle

```diff
@@ -65,11 +65,11 @@ dependencies {
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jackson}"
 
     // Azure related dependencies
-    implementation 'com.azure:azure-identity:1.2.5'
-    implementation 'com.azure.resourcemanager:azure-resourcemanager:2.4.0'
-    implementation 'com.azure:azure-storage-common:12.11.1'
-    implementation 'com.azure:azure-storage-file-datalake:12.5.1'
-    implementation 'com.azure:azure-data-tables:12.1.0'
+    implementation 'com.azure:azure-identity:1.9.1'
+    implementation 'com.azure.resourcemanager:azure-resourcemanager:2.28.0'
```

Comment:
Someday we should switch TDR to use gradle conventions subprojects so you don't have to repeat dependency versions across files.

Reply:
Yeah...I agree...or just remove this entire subproject altogether